group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/RangeAssignorTest.java

```diff
@@ -687,7 +688,7 @@ public void testReassignmentWhenMultipleSubscriptionsRemovedAfterInitialAssignme
             currentAssignmentForC
         ));
 
-        AssignmentSpec assignmentSpec = new AssignmentSpec(members);
+        AssignmentSpec assignmentSpec = new AssignmentSpec(members, HETEROGENEOUS);
```

Comment:
Is this change enough to cover the `HETEROGENEOUS` path?

Reply:
Yeah it covers it. Nothing changed in the hetero path anyways